docs/how-to/uc-image-creation.rst

```diff
@@ -28,12 +28,24 @@ Create the model assertion
 The model assertion is a digitally signed document that describes the content of the Ubuntu Core image.
 Read the `model assertion`_ documentation before continuing.
 
-Below is an example model assertion in YAML, describing a `core22` Ubuntu Core
-image:
+Below are example model assertions, describing the Ubuntu Core image content for most recent releases:
 
-.. literalinclude:: uc-image-creation/model.json
-   :language: json
-   :emphasize-lines: 4, 8-11, 19-24
+.. tabs::
```

Comment:
Also a nitpick.

But I would suggest putting the most recent release as the first tab. Since for software we typically want customers to upgrade to the latest greatest release.